src/js/matomo.js

```diff
@@ -63,6 +64,12 @@ function getBrowserSize() {
         _paq.push(['addTracker', `${u}tracker.php`, drupalSettings.matomo_site_id]);
       }
 
+      // If the site is Etusivu-instance and there are newsTaxonomyTermIds set, sent them to custom dimension.
+      if (['141', '1292'].includes(drupalSettings.matomo_site_id.toString()) && newsTaxonomyTermIds()) {
```

Comment:
Might be better to put `['141', '1292'].includes(drupalSettings.matomo_site_id.toString()` in  a variable since we use it twice